### ISSUE TITLE
csv_weights cannot be set

### ISSUE THREAD

**Issue by tigerhawkvok** [aws/sagemaker-python-sdk#1135]
Please fill out the form below.

### System Information
- **Framework (e.g. TensorFlow) / Algorithm (e.g. KMeans)**: XGBoost
- **Framework Version**: 0.90
- **Python Version**: 3
- **CPU or GPU**:
- **Python SDK Version**: Latest
- **Are you using a custom image**:

### Describe the problem
Can't establish an XGBoost estimator with `csv_weights` set to 1, as per https://docs.aws.amazon.com/sagemaker/latest/dg/xgboost.html : 

> To differentiate the importance of labelled data points use Instance Weight Supports
>
> Amazon SageMaker XGBoost allows customers to differentiate the importance of labelled data points by assigning each instance a weight value. For text/libsvm input, customers can assign weight values to data instances by attaching them after the labels. For example, label:weight idx_0:val_0 idx_1:val_1.... For text/csv input, customers need to turn on the csv_weights flag in the parameters and attach weight values in the column after labels. For example: label,weight,val_0,val_1,...).

Now, the docs don't say where the `XGBoost` class takes the argument, so I tried the obvious locations which all failed.

### Minimal repro / logs

As an estimator paramter:

![image](https://user-images.githubusercontent.com/165937/69581142-d9ab3100-0f8a-11ea-8471-d7ad13eff52d.png)

As a float in the estimator fitting: 

![image](https://user-images.githubusercontent.com/165937/69581161-e596f300-0f8a-11ea-85f8-ef468b618dd1.png)

As a string in the estimator fitting:

![image](https://user-images.githubusercontent.com/165937/69581187-efb8f180-0f8a-11ea-8424-8eb7c8bff561.png)

As a fit parameter:

![image](https://user-images.githubusercontent.com/165937/69581246-07907580-0f8b-11ea-8471-6176e325656a.png)

**Comment by ChoiByungWook** [comment]
Reference: 0420645671
